dist/stat_data.py

```diff
@@ -998,6 +999,7 @@ def __init__(self, name, desc, flags=''):
     CacheStat('cache_hs_write_squash', 'history store table writes requiring squashed modifies'),
     CacheStat('cache_inmem_split', 'in-memory page splits'),
     CacheStat('cache_inmem_splittable', 'in-memory page passed criteria to be split'),
+    CacheStat('cache_eviction_blocked_multi_block_reconcilation_during_checkpoint', 'multiple block reconciliation whilst checkpoint is running'),
```

Comment:
multi-block reconciliation blocked whilst checkpoint is running

Reply:
fixed https://github.com/wiredtiger/wiredtiger/pull/10046/commits/c11ee27e165685f470eb81ca1e7890b66b305db7